remove c_nonce and c_nonce_expires_in from credential endpoint response (#404)
openid/OpenID4VCI: openid-4-verifiable-credential-issuance-1_0.md

  * `invalid_credential_request`: The Credential Request is missing a required parameter, includes an unsupported parameter or parameter value, repeats the same parameter, or is otherwise malformed.
  * `unsupported_credential_type`: Requested Credential type is not supported.
  * `unsupported_credential_format`: Requested Credential Format is not supported.

>   * `invalid_proof`: The `proof` or `proofs` parameter in the Credential Request is invalid: (1) if both fields are missing, or (2) both are present simultaneously, or (3) one of the provided key proofs is invalid, or (4) if a `c_nonce` was previously provided and at least one of the key proofs is not linked to a valid `c_nonce` value (refer to (#nonce-response)).

Reviewer: agree very much, distinguishing this error, makes it clear for the wallet what to do

>   * `invalid_proof`: The `proof` or `proofs` parameter in the Credential Request is invalid: (1) if both fields are missing, or (2) both are present simultaneously, or (3) one of the provided key proofs is invalid, or (4) if a `c_nonce` was previously provided and at least one of the key proofs is not linked to a valid `c_nonce` value (refer to (#nonce-response)).

Author: Thanks @c2bo, i also thought about a specific error code and like your suggestion

  * `invalid_credential_request`: The Credential Request is missing a required parameter, includes an unsupported parameter or parameter value, repeats the same parameter, or is otherwise malformed.
  * `unsupported_credential_type`: Requested Credential type is not supported.
  * `unsupported_credential_format`: Requested Credential Format is not supported.

>   * `invalid_proof`: The `proof` or `proofs` parameter in the Credential Request is invalid: (1) if both fields are missing, or (2) both are present simultaneously, or (3) one of the provided key proofs is invalid, or (4) if at least one of the key proofs is not linked to a required `c_nonce` value (refer to (#nonce-response)).

Reviewer 2: ```suggestion
  * `invalid_proof`: The `proof` or `proofs` parameter in the Credential Request is invalid: (1) if both fields are missing, or (2) both are present simultaneously, or (3) one of the provided key proofs is invalid, or (4) if at least one of the key proofs does not contain a `c_nonce` value (refer to (#nonce-response)).
```

>   * `invalid_nonce`: The `proof` or `proofs` parameter in the Credential Request uses an invalid nonce: at least one of the key proofs is not linked to a valid `c_nonce` value. The wallet should retrieve a new `c_nonce` value (refer to (#nonce-endpoint)).

Reviewer 2: If we have the `invalid nonce` error would it make sense to use it also for the `invalid_proof` case (4)?

>   * `invalid_nonce`: The `proof` or `proofs` parameter in the Credential Request uses an invalid nonce: at least one of the key proofs is not linked to a valid `c_nonce` value. The wallet should retrieve a new `c_nonce` value (refer to (#nonce-endpoint)).

Reviewer 2: ```suggestion
  * `invalid_nonce`: The `proof` or `proofs` parameter in the Credential Request uses an invalid nonce: at least one of the key proofs does not contain a valid `c_nonce` value. The wallet should retrieve a new `c_nonce` value (refer to (#nonce-endpoint)).
```
